packages/manager/modules/pci/src/components/pci-order-command/translations/Messages_pt_PT.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
**thought:** Better to use `git mv` if we just have to move translations.

Reply:
Noted